packages/anoncreds/src/formats/LegacyIndyCredentialFormatService.ts

```diff
@@ -399,6 +399,7 @@ export class LegacyIndyCredentialFormatService implements CredentialFormatServic
       })
       credentialRecord.setTags({
         anonCredsRevocationRegistryId: credential.revocationRegistryId,
+        anonCredsUnqualifiedRevocationRegistryId: anonCredsCredential.rev_reg_id,
```

Comment:
Is this addition also needed in the `acceptRequest` method? (Line 277)

Reply:
That's for the issuer part, but yes, it should be added also here if we follow this approach.